discord/webhook.py

```diff
@@ -642,6 +652,62 @@ def edit(self, **kwargs):
 
         return self._adapter.edit_webhook(**payload)
 
+    async def _async_move(self, channel_id, token, payload):
+        await self._adapter.move_webhook(token, **payload)
+        self.channel_id = channel_id
+
+    def _sync_move(self, channel_id, token, payload):
+        self._adapter.move_webhook(token, **payload)
+        self.channel_id = channel_id
+
+    def move_to(self, channel):
+        """|maybecoro|
+
+        Moves this Webhook.
```

Comment:
Moving webhook is not a discord term (Moving members is) , so `Moves this Webhook to other channel.` may be better.